packages/web-app-admin-settings/src/components/Users/UsersList.vue

```diff
@@ -7,6 +7,30 @@
       :label="$gettext('Filter users')"
       autocomplete="off"
     />
+    <div class="oc-flex oc-flex-middle oc-ml-m oc-mb-m oc-mt-m">
+      <div class="oc-mr-m oc-flex oc-flex-middle">
+        <oc-icon name="filter-2" fill-type="line" />
+        <span v-text="$gettext('Filter:')" />
+      </div>
+
+      <!-- TODO: just for demo purposes, remove later -->
+      <item-filter
```

Comment:
Really good to have it for the code review! 👍  Please remove again before merge 😉 (I guess that's what you meant by `remove later`, just to be sure)